Knoema.Client/Client.cs

```diff
@@ -36,22 +38,29 @@ public class Client
 
 		public int HttpTimeout { get; set; }
 
-		private Client()
+		private Client(bool ignoreCertErrors, string scheme)
```

Comment:
Why do you need IgnoreCertErrors?
Why do you need scheme?

Reply:
these options allows to use API via HTTPS with invalid certificate

Reply:
I understand what it allows to do. but why do you need this? What is the reason? Because it looks like you are trying to workaround the server issue here